src/NLog/Layouts/SimpleLayout.cs

```diff
@@ -263,14 +263,15 @@ protected override string GetFormattedMessage(LogEventInfo logEvent)
                 }
                 catch (Exception exception)
                 {
-                    if (exception.MustBeRethrown())
+                    //also check IsErrorEnabled, otherwise 'MustBeRethrown' writes it to Error
+                    if (InternalLogger.IsWarnEnabled || InternalLogger.IsErrorEnabled)
```

Comment:
I is for backwardscompability. 

If I remove `InternalLogger.IsErrorEnabled`, the error will be written to ERROR level instead of WARN. This is a unneeded breaking change.